<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GeoDataPointmap_PointsManager_FilterSelectedPointsDialog</class>
 <widget class="QDialog" name="GeoDataPointmap_PointsManager_FilterSelectedPointsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>343</width>
    <height>270</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Filter Selected Points</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="propertiesGroupBox">
     <property name="title">
      <string>Properties of selected points</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="numPointsLabel">
        <property name="text">
         <string>Number of points:</string>
        </property>
        <property name="buddy">
         <cstring>numPointsValueLabel</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="numPointsValueLabel">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="minPointLabel">
        <property name="text">
         <string>Minimum value:</string>
        </property>
        <property name="buddy">
         <cstring>minValueLabel</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="RealNumberEditWidget" name="minValueLabel">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="maxPointLabel">
        <property name="text">
         <string>Maximum value:</string>
        </property>
        <property name="buddy">
         <cstring>maxValueLabel</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="RealNumberEditWidget" name="maxValueLabel">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="conditioinGroupBox">
     <property name="title">
      <string>Filtering condition</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QRadioButton" name="greaterRadioButton">
        <property name="text">
         <string>Select points with value greater than threshold</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="lessRadioButton">
        <property name="text">
         <string>Select points with value less than threshold</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="thresholdLabel">
       <property name="text">
        <string>Threshold value:</string>
       </property>
       <property name="buddy">
        <cstring>thresholdEdit</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="RealNumberEditWidget" name="thresholdEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>6</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Apply|QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>RealNumberEditWidget</class>
   <extends>QLineEdit</extends>
   <header>guibase/widget/realnumbereditwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>GeoDataPointmap_PointsManager_FilterSelectedPointsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>GeoDataPointmap_PointsManager_FilterSelectedPointsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
